JpegXmpWritePluginMDE/JpegXmpWritePluginMDE.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="JetBrains.Annotations"/>
```

Comment:
Sorry, missed this one before - could we add a ```PrivateAssets="All"``` to the PackageReference?

e.g. 
![image](https://github.com/ahsanaman92/JpegXmpWritePluginMDE/assets/1178570/e1296a56-2569-4e9b-99b9-2eaf540732d9)

(Otherwise, it might cause callers to gain a dependency on that package when they don't need to)

Reply:
Thank you for catching that. Wasn't aware of the significance of this flag. Done